core/datastore/src/test/java/com/threegap/bitnagil/datastore/storage/AuthTokenDataStoreImplTest.kt

```
@@ -0,0 +1,231 @@
+    private lateinit var authTokenDataStore: AuthTokenDataStore
+
+    private object FakeAuthTokenSerializer : Serializer<AuthToken> {
+        override val defaultValue: AuthToken
+            get() = AuthToken()
+
+        override suspend fun readFrom(input: InputStream): AuthToken {
+            return try {
+                input.bufferedReader().use {
+                    Json.decodeFromString(AuthToken.serializer(), it.readText())
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)